_layouts/default.html

```diff
@@ -81,7 +81,7 @@
               <div class="dropdown-menu" aria-labelledby="nav-about-dropdown">
                 <a class="dropdown-item" href="https://datacommons.org/about">About Data
                   Commons</a>
-                <a class="dropdown-item" href="/blog.html">Blog</a>
+                <a class="dropdown-item" href="https://blogs.datacommons.org/">Blog</a>
```

Comment:
Should this be blog.datacommons.org? (no s)

Reply:
great catch - thank you!